README.md

```diff
@@ -71,6 +71,22 @@ host system for storage, pass `-v/host-path:/data`.
 
 </details>
 
+### Docker Compose
+
+**1. Configure the default secrets as environment variables**
+
+```yaml
+environment:
+  RGW_DEFAULT_USER_ACCESS_KEY: "test"
+  RGW_DEFAULT_USER_SECRET_KEY: "test"
+```
+
+**2. Run docker-compose**
+
+```bash
+docker-compose up -d
```

Comment:
`-d` detaches the process from the shell (aka. daemonizes the process), so you'll be dropped back to your shell while the docker-compose process continues in the background. This allows you to log out of the machine without killing the process or run other continue using the shell with other commands. It's entirely optional.
You can use `docker-compose down` to stop the containers later on.

Reply:
What i mean is `docker-compose` vs `docker compose`. On my Ubuntu installation i get 
```
$ which docker-compose                         
docker-compose not found
```

Regarding the `-d`. I prefer to not mention it in the docs because running it in front helps to watch what is going on. User that want to run it in background can simply add the `-d` on their own.